modules/ocf_printhost/files/ocfps

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Use the user command pdftops, not the CUPS filter.
+PDFTOPS=/usr/bin/pdftops
+
+if [ "$#" -ne 5 ] && [ "$#" -ne 6 ]; then
+    echo "Usage: $0 job-id user title copies options [file]"
+    exit
+fi
+
+if [ "$#" -eq 5 ]; then
+    $PDFTOPS - - | pstops "$@"
+else
+    $PDFTOPS "$6" - | pstops "${@:1:5}"
+fi
```

Comment:
While I agree this stuff should be in a comment on the top, I was planning for our documentation to briefly explain the basics of CUPS architecture as well as our own setup, since the online documentation on CUPS is severely lacking (our setup makes no sense unless you understand CUPS somewhat anyways). I myself believe it would be helpful for OCFers and the general public to state the call signature and location of custom CUPS filters as part of an overview.

On the other hand, it is true that those details are unnecessary to the basic understanding of our setup, since they only matter within this file. There is an argument to be made that OCFers should just read the comments in the script to learn how it's supposed to work.